index.html

```diff
@@ -73,6 +76,196 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
           </ul>
         </div>
       </section>
+
+      <!-- Work section -->
+      <section class="work">
+        <!-- work card 1 -->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio1.png" alt="Project 1" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Tonic</h1>
```

Comment:
- [ ] For the semantic purpose only one h1 is allowed on the page. Kindly make sure you use another `header tag`
`Hint: You may consider using h2..6`